src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -622,6 +622,12 @@ const terminalConfiguration: IConfigurationNode = {
 				localize('terminal.integrated.focusAfterRun.none', "Do nothing."),
 			]
 		},
+		[TerminalSettingId.InitialHint]: {
+			restricted: true,
+			markdownDescription: localize('terminal.integrated.initialHint', "Controls if the first terminal without input will show a hint about available actions when it is focused."),
+			type: 'boolean',
+			default: true
+		},
```

Comment:
Settings now live in the contribs, see `terminalAccessibilityConfiguration` example below. Same with the interface in terminal.ts